bokehjs/src/coffee/models/layouts/layout_dom.coffee

```diff
@@ -207,26 +207,26 @@ class LayoutDOM extends Model
       'whitespace-left' : @_whitespace_left
       'whitespace-right' : @_whitespace_right
     }
-    if @responsive is 'stretch_both'
+    if @sizing_mode is 'stretch_both'
       constrained_variables = _.extend(constrained_variables, {
         'width': @_width
         'height': @_height
       })
-    if @responsive is 'scale_width'
+    if @sizing_mode is 'scale_width'
       constrained_variables = _.extend(constrained_variables, {
         'width': @_width
       })
-    if @responsive is 'scale_height'
+    if @sizing_mode is 'scale_height'
       constrained_variables = _.extend(constrained_variables, {
         'height': @_height
       })
     return constrained_variables
 
   @define {
-      height:   [ p.Number, null ]
-      width:    [ p.Number, null ]
-      disabled: [ p.Bool, false ]
-      responsive: [ p.Responsive, null ]
+      height:      [ p.Number              ]
+      width:       [ p.Number              ]
+      disabled:    [ p.Bool,       false   ]
+      sizing_mode: [ p.SizingMode, "fixed" ]
```

Comment:
I changed it here to fix those defaults inconsistencies that cropped up. Also I don't understand what a `None` / `null` value is supposed to mean? I feel like it should not be allowed at all